modules/build/src/main/scala/scala/build/options/ScalaOptions.scala

```diff
@@ -13,8 +13,14 @@ final case class ScalaOptions(
   extraScalaVersions: Set[String] = Set.empty,
   compilerPlugins: Seq[Positioned[AnyDependency]] = Nil,
   platform: Option[Platform] = None,
-  extraPlatforms: Set[Platform] = Set.empty
+  extraPlatforms: Set[Platform] = Set.empty,
+  supportedScalaVersionsUrl: Option[String] = None
 ) {
+
+  lazy val scalaVersionsUrl = supportedScalaVersionsUrl.getOrElse(
+    "https://raw.githubusercontent.com/VirtusLab/scala-cli/master/scala-version.scala"
```

Comment:
Maybe we should put that file in a separate repo, add a version suffix to it, and give it a `.json` extension, like
```suggestion
    "https://raw.githubusercontent.com/VirtusLab/scala-cli-scala-versions/master/scala-versions-v1.json"
```

Reply:
cc @romanowski

Reply:
I am fine with that.